src/network/MissionControlProtocol.cpp

```diff
@@ -221,6 +224,25 @@ void MissionControlProtocol::sendRoverPos() {
 	}
 }
 
+static bool validateWaypointNavRequest(const json& j) {
+	return util::validateKey(j, "latitude", val_t::number_unsigned) &&
+		   util::validateKey(j, "longitude", val_t::number_unsigned) &&
```

Comment:
latitude and longitude aren't unsigned ints, this should probably be `val_t::number_float`?

Reply:
It only validates if they are number_unsigned. I battled it for like an hour. I saw theyre floats in MissionControl but it won't validate as floats. If you have an idea that'd be great.